src/database.cpp

```
@@ -129,142 +180,243 @@ static bool executeQuery(tfs::detail::Mysql_ptr& handle, std::string_view query,
+			q.remove_prefix(1);
+		}
+
+		auto wordEnd = q.find(' ');
+		std::string_view firstWord = (wordEnd == std::string_view::npos) ? q : q.substr(0, wordEnd);
+
+		char upper[9];
+		size_t len = std::min(firstWord.size(), sizeof(upper));
+		for (size_t i = 0; i < len; ++i) {
+			upper[i] = static_cast<char>(std::toupper(static_cast<unsigned char>(firstWord[i])));
+		}
+		std::string_view cmd{upper, len};
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)